var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -52,7 +52,7 @@ class PyCython(PythonPackage):
     depends_on("cxx", type="build")  # generated
 
     # https://github.com/cython/cython/issues/5751 (distutils not yet dropped)
-    depends_on("python@:3.11", type=("build", "link", "run"))
+    depends_on("python@:3.12", type=("build", "link", "run"))
```

Comment:
Is 3.13 not yet supported? Is this true for all versions of cython?

Reply:
Python 3.13 was not supported when I wrote this. Python 3.13 has been added in Cython 3.1.0a1
https://github.com/cython/cython/releases/tag/3.1.0a1